### PR TITLE
adding vector policy

### PR DESCRIPTION
**Pull request by vipulvishal-ms**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py [hunk 1]
```diff
@@ -208,8 +214,9 @@ def load_arguments(self, _):
...
-        c.argument('indexing_policy', options_list=['--idx'], type=shell_safe_json_parse, completer=FilesCompleter(), help='Indexing Policy, you can enter it as a string or as a file, e.g., --idx @policy-file.json or ' + SQL_GREMLIN_INDEXING_POLICY_EXAMPLE)
+        c.argument('indexing_policy', options_list=['--idx'], type=shell_safe_json_parse, completer=FilesCompleter(), help='Indexing Policy, you can enter it as a string or as a file, e.g., --idx @policy-file.json or ' + SQL_INDEXING_POLICY_EXAMPLE)
         c.argument('client_encryption_policy', options_list=['--cep'], type=shell_safe_json_parse, completer=FilesCompleter(), validator=validate_client_encryption_policy, help='Client Encryption Policy, you can enter it as a string or as a file, e.g., --cep @policy-file.json or ' + SQL_CLIENT_ENCRYPTION_POLICY_EXAMPLE)
+        c.argument('vector_embedding_policy', options_list=['--vector-embedding-policy'], type=shell_safe_json_parse, completer=FilesCompleter(), help='Vector Encryption Policy, you can enter it as a string or as a file, e.g., --vector-embedding-policy @policy-file.json or ' + SQL_VECTOR_EMBEDDING_POLICY_EXAMPLE)
```
Comment (kr-santosh) [inline src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py:219]: so we are just giving one options --vector-embedding-policy and no other options -vep

File: src/azure-cli/azure/cli/command_modules/cosmosdb/tests/latest/test_cosmosdb_commands.py [hunk 2]
```diff
@@ -572,23 +573,26 @@ def test_cosmosdb_sql_container(self, resource_group):
             'nttl': new_default_ttl,
             'unique_key': unique_key_policy,
             "conflict_resolution": conflict_resolution_policy,
-            "indexing": indexing
+            "indexing": indexing,
+            "vector_embedding": vector_embedding_policy
         })
 
-        self.cmd('az cosmosdb create -n {acc} -g {rg}')
+        self.cmd('az cosmosdb create -n {acc} -g {rg} --capabilities EnableNoSQLVectorSearch')
```
Comment (kr-santosh) [inline src/azure-cli/azure/cli/command_modules/cosmosdb/tests/latest/test_cosmosdb_commands.py:580]: is this required?

**Review by kr-santosh — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py (hunk 1 above)
Comment (vipulvishal-ms) [reply src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py:219]: yes, one option

File: src/azure-cli/azure/cli/command_modules/cosmosdb/tests/latest/test_cosmosdb_commands.py (hunk 2 above)
Comment (vipulvishal-ms) [reply src/azure-cli/azure/cli/command_modules/cosmosdb/tests/latest/test_cosmosdb_commands.py:580]: ya, else i get capability not enabled on account

**Pull request merged**